lib/Connection.php

```diff
@@ -335,6 +344,18 @@ public function query($sql, &$values=array())
 				throw new DatabaseException($this);
 		} catch (PDOException $e) {
 			throw new DatabaseException($e);
+		} finally {
```

Comment:
`finally` isn't available until PHP 5.5, and this project currently supports PHP 5.3. So we'd either have to change how we perform this task, or bump the required version in the `composer.json`.

That being said, I think that this task is likely better handled outside of a `finally` block anyway, although it might make the `total_elapsed_query_time` harder to properly support.